config/checkstyle-examples-checks.xml

```diff
@@ -14,6 +14,15 @@
   </module>
 
   <!-- Regexp -->
+  <module name="RegexpMultiline">
+    <property name="id" value="exampleFileXmlConfigPresence"/>
+    <property name="format" value="\/*xml\n&lt;module name=&quot;Checker&quot;&gt;"/>
+    <property name="minimum" value="1"/>
+    <property name="maximum" value="1"/>
+    <property name="matchAcrossLines" value="true"/>
+    <property name="message"
+              value="Example files must start with /*xml and have Checker module declared on top"/>
```

Comment:
I don't see why we need to enforce checker, if config is bad, loading of config will fail.

Reply:
Yes, not required